db/postgres/postgres.go

```diff
@@ -0,0 +1,59 @@
+package postgres
+
+import (
+	"database/sql"
+	"fmt"
+
+	"github.com/lib/pq"
+
+	dbType "digitalis.io/vals-operator/db/types"
+	ctrl "sigs.k8s.io/controller-runtime"
+)
+
+func runPostgresQuery(dbQuery dbType.DatabaseQuery, host string) error {
+	psqlconn := fmt.Sprintf("host=%s port=%d user=%s password=%s dbname=%s connect_timeout=5 sslmode=disable",
```

Comment:
sslmode `allow` or `prefer` would work better in case a server insists on using ssl

Reply:
It appears the Go library does not support `prefer`. I have kept it as `disable` but I'm allowing the user to set the environment variable `PGSSLMODE` to override. This will be a global setting and not a per DB unfortunately. We will add full TLS support to the next version.